chain/lf3/config.go

```diff
@@ -85,34 +68,27 @@ func NewConfig(nn dtypes.NetworkName) *Config {
 	if nn == "testnetnet" {
 		nn = "filecoin"
 	}
-	pollInterval := 15 * time.Minute
-	if envVar := os.Getenv("LOTUS_F3_POLL_INTERVAL"); len(envVar) != 0 {
-		d, err := time.ParseDuration(envVar)
-		if err != nil {
-			log.Errorf("invalid duration in LOTUS_F3_POLL_INTERVAL, defaulting to %v", pollInterval)
-		} else {
-			pollInterval = d
-		}
-
-	}
 	c := &Config{
-		BaseNetworkName:          gpbft.NetworkName(nn),
-		PrioritizeStaticManifest: true,
-		DynamicManifestProvider:  buildconstants.F3ManifestServerID,
-		AllowDynamicFinalize:     false,
-		ContractPollInterval:     pollInterval,
-	}
-	if buildconstants.F3BootstrapEpoch >= 0 {
-		c.StaticManifest = NewManifest(
-			c.BaseNetworkName,
-			policy.ChainFinality,
-			buildconstants.F3BootstrapEpoch,
-			time.Duration(buildconstants.BlockDelaySecs)*time.Second,
-			buildconstants.F3InitialPowerTableCID,
-		)
+		BaseNetworkName: gpbft.NetworkName(nn),
+		StaticManifest:  buildconstants.F3Manifest(),
 	}
-	if buildconstants.F3ParamsAddress != "" {
-		c.ContractAddress = buildconstants.F3ParamsAddress
+	c.StaticManifest = buildconstants.F3Manifest()
+	if c.StaticManifest != nil {
+		if ptCid := os.Getenv("F3_INITIAL_POWERTABLE_CID"); ptCid != "" {
```

Comment:
Oh I see this logic is moved here. 

I would probably encapsulate it in `buildconstants.F3Manifest` just to keep all funky env vars as close to one place as possible?